conman/routes/models.py

```diff
@@ -74,6 +84,64 @@ def handle(self, request, path):
         # Deal with the request
         return handler.handle(request, path)
 
+    def move_to(self, new_url, *, move_children):
```

Comment:
I suspect either default might lead to cases of unexpected pain, so I thought it might be best to force the programmer to make a decision one way or the other, given the naming doesn't clearly suggest a default. Do you think this might be clunky as is?